tests/test_iterations.py

```diff
@@ -230,7 +233,14 @@ def test_iter_nonmc_acquisition_function(campaign, n_iterations, batch_size):
     "kernel", valid_kernels, ids=[c.__class__ for c in valid_kernels]
 )
 @pytest.mark.parametrize("n_iterations", [3], ids=["i3"])
-def test_iter_kernel(campaign, n_iterations, batch_size):
+def test_kernels(campaign, n_iterations, batch_size):
+    run_iterations(campaign, n_iterations, batch_size)
+
+
+@pytest.mark.parametrize("kernel", valid_kernel_factories)
+@pytest.mark.parametrize("n_iterations", [3], ids=["i3"])
+def test_kernel_factories(campaign, n_iterations, batch_size):
+    print(campaign.recommender)
```

Comment:
Just out of curiosity, why is the recommender being printed here?

Reply:
leftover, removed